add-on/_locales/en/messages.json

```diff
@@ -351,6 +351,14 @@
     "message": "Turn plaintext /ipfs/ paths into clickable links",
     "description": "An option description on the Preferences screen (option_linkify_description)"
   },
+  "option_webuiFromDNSLink_title": {
+    "message": "Load the latest Web UI",
```

Comment:
@olizilla would appreciate your thoughts on how to name and describe this feature.
My initial stab looks like this:

> ![2019-07-18--22-48-30](https://user-images.githubusercontent.com/157609/61490868-3465c800-a9ae-11e9-9108-9ed26593f43c.png)